web/api/v1/api_test.go

```diff
@@ -539,6 +540,7 @@ func testEndpoints(t *testing.T, api *API, testLabelAPI bool) {
 		query    url.Values
 		response interface{}
 		errType  errorType
+		sorter   func(interface{}) interface{}
```

Comment:
A list of interfaces would be more in line with the sort package, and have it not return the result given that it mutates.

Reply:
I did consider that - the only problem with that it wouldn't match the response type we have right now resulting in a much bigger diff. Happy to amend it if you think it's still worth it!

Reply:
I have fixed the latter though.